### ISSUE TITLE
Starships (Just the Beginning)

### ISSUE THREAD

**Issue by Swampson** [queryluke/masseffect-5e#159]
It's safe to assume that the majority of campaigns using this setting are going to involve the PCs having access to a rad spaceship. So we will need to create rules for bitchin' starblasters. This is just the very first, very rough outline:

- **Expensive:** these things are going to cost so much that there's no way a group of beginning adventurers would buy their own. They will need to be issued, inherited, or stolen.
- **Ship's Systems:** All ships will have "systems" which are basically skills, but for ships. Not all ships will have all of these systems. **Bold** systems are the ones that will be common to all ships. [Bracketed] terms are the PC skills that can be used to operate these systems more effectively
  - **Helm**: piloting the ship [Piloting]
  - **Navigation**: plotting a course to a destination [Navigation]
  - Weapons: pew pew [Gunnery]
  - Shields: wep wep [Engineering]
  - Cyber-warfare: computer systems intrusion and defense [Hacking]
  - Recreation: this could be a lounge, bar, game room, etc that may have an impact on crew morale on larger vessels where an NPC crew may be present [Could fall under several skills, but Performance by default]
  - **Power Core**: make ship zoom [Engineering]
  - Specialty Systems: a mining array, probe-launcher, stealth drive, science lab, live storage of livestock or scientific specimens, guest quarters for paying passengers, etc
- **VI**: in order to make it easier to get a campaign (and ship) up-and-running, we can assume any spacecraft of adventuring-party size or smaller is equipped with a VI that can automate normal functioning of all built-in systems. In game terms, this means that whenever there's a ship-skill roll is required and no PC or NPC has the appropriate skill, the ship itself adds a +2 to the roll. The VI itself may be a ship's system and represent a ship-wide proficiency bonus. So all ships have at least a Lvl 1 VI, but Level 5(+3), 9(+4), 13(+5), and 17(+6) VIs could be purchased/upgraded. A level 17 VI may actually be an AI, like EDI.
- **Naval Combat** ship-to-ship combat could be run in one of two ways: either everyone rolls for initiative and combat proceeds in the normal order, or each ship rolls initiative and all crew members (PCs and NPCs) act in unison on their ship's initiative order. A character's "move" action can be used to move between different parts of the ship. The DM should abstract this as-necessary: it would only take a single turn to move from the bridge to the CIC on-board the Normandy, but it may take two terms to get from the CIC to the med-bay. On larger ships it could take several turns to move between points of interest. Because space combat is rarely a sudden ambush, it should generally be assumed that enough time is given for a call to battle stations and that all characters can get to their assigned stations before combat really begins.
- **Ship Stats**: Ships should have:
  - HP
  - SP
  - AC
  - "Speed" - this should be an abstract stat that stands-in for a ship's speed, maneuverability, etc, and should probably not be a number tied to a real unit of measurement

**Comment by queryluke** [comment]
Another person brought this idea up in #59 (which I'll close since they never followed up with their ideas). I like the idea, but would want to package it as an add-on. I.e., something that players can choose to incorporate into their campaign, but not something that is required for every campaign.

**Comment by Swampson** [comment]
Yeah, this would definitely be a stand-alone module. I’ll work on filling it out more.

**Comment by Swampson** [comment]
My initial thought for the way this would work is that I would map some or all of the standard 18 skills to shipboard counterparts, so that players would be able to roll-play their characters controlling various ship systems when applicable. However, I think is a little messy at best and nonsensical at worst.

Instead, I'm leaning toward treating ship systems as tool proficiencies. When we incorporate Starship rules into the PHB, I'll include a variant rule that GMs can choose to employ where each PC gets to pick a single ship system proficiency if it makes sense for the campaign. 

Whenever rolling for a ship's system ("System Checks"), each _player_ will be responsible for rolling the dice for one or more systems when they come up. If their PC is proficient in that system, they'll be RPing as their PC operating that system. If the PC is not proficient, the player will be RPing as an NPC crew member or maybe even as the VI or one of its subroutines. The goal here is to keep all of the players involved and engaged during starship encounters, even if their PC wouldn't realistically be involved in the events.

**Comment by bpmcpherson** [comment]
Is this discussion going to also include land-based vehicles (i.e. Mako, Hammerhead, Nomad, etc.)?

**Comment by Swampson** [comment]
I think that land vehicles should probably have their own discussion because there's enough distinction between starships and ground vehicles.

**Comment by bpmcpherson** [comment]
Alright, sounds good to me.
